samples/SampleApp/Startup.cs

```diff
@@ -40,6 +41,12 @@ public static void Main(string[] args)
                 Console.WriteLine("Unobserved exception: {0}", e.Exception);
             };
 
+            var configuration = new ConfigurationBuilder()
+                .AddEnvironmentVariables()
+                .Build();
+            if (!ushort.TryParse(configuration["BASE_PORT"], System.Globalization.NumberStyles.None, System.Globalization.CultureInfo.InvariantCulture, out var basePort))
```

Comment:
Style nits:

- Add new line before the `if`
- Use a block `{ }`
- `using System.Globalization;`